WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -1421,7 +1365,7 @@ public void onAutomatedTransferInitiated(OnAutomatedTransferInitiated event) {
      * Unfortunately we can't close the progress dialog until both the site and its plugins are fetched. Check out
      * `onSiteChanged` for the next step.
      */
-    @SuppressWarnings("unused")
+    @SuppressWarnings("unused") // Progress Dialog Deprecation on API 26
```

Comment:
Minor (🔍): Consider removing this extra comment here ([context](https://github.com/wordpress-mobile/WordPress-Android/pull/20968#discussion_r1637463943)).

Reply:
Done!